ast_canopy/tests/test_parse_from_source.py

```diff
@@ -246,7 +260,9 @@ def test_load_ast_typedefs(sample_typedef_source, test_pickle):
     assert len(typedefs) == 3
 
     assert structs[0].name == "A"
-    second_struct_name = structs[1].name  # Originally unnamed, renamed with its AST ID.
+    second_struct_name = structs[
+        1
+    ].name  # Originally unnamed, renamed with its AST ID.
```

Comment:
Oof, I imagine something like this require moving the comment to a separate line to fix.